sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_base.json

```diff
@@ -479,11 +498,37 @@
                 "description": "Feature store online store subscription id"
             }
         },
-        "materialization_identity_resource_id": {
+        "materializationIdentityOption": {
             "type": "string",
-            "defaultValue": "",
+            "defaultValue": "none",
+            "allowedValues": [
+                "new",
+                "existing",
+                "none"
```

Comment:
"none" means we do not provision an UAI?

Reply:
yes